packages/angular-output-target/src/types.ts

```diff
@@ -18,6 +18,10 @@ export interface OutputTargetAngular {
   excludeComponents?: string[];
   includeImportCustomElements?: boolean;
   customElementsDir?: string;
+  /**
+   * `true` to generate a single component Angular module for each component.
+   */
+  createAngularModules?: boolean;
```

Comment:
I wonder if it makes sense to rename this to be a little more specific to single-component modules. Thoughts on renaming this to 
```suggestion
  createSingleComponentAngularModules?: boolean;
```